### PR TITLE
doc: update IBD requirements in doc/README.md

### PR DESCRIPTION
**Pull request by Mackain**
A small change to the first paragraph of the Setup part of the README that has been bugging me for a while. 
The disk space required for the Bitcoin transactions can no longer be described as "a few" hundred gigabytes.
So I thought it was time it was changed to "several" instead.

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Bitcoin Core
 
 Setup
 ---------------------
-Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions, which requires a few hundred gigabytes of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
+Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions, which requires several hundred gigabytes of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
```
Comment (jonatack) [inline doc/README.md:6]: This change looks like a continuation of https://github.com/bitcoin/bitcoin/pull/14511.

Perhaps go a bit further, if we continue to prefer to give an idea of disk space and sync time here.

```suggestion
Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions, which requires several hundred gigabytes or more of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to several days or more.
```

Comment (Mackain) [reply doc/README.md]: @jonatack good point! thanks for the suggestion

**Review by IsaacCavallaro — APPROVED** [review]

**Review by jonatack — COMMENTED** [review]
ACK 3208df2a100f58569165081cd20c02abed827286

Might be good to update the PR description to reflect the changes (and the commit message as well, if you need to repush).

Suggestion for the PR title: `doc: update IBD requirements in doc/README.md`

**Comment by Mackain** [comment]
> ACK [3208df2](https://github.com/bitcoin/bitcoin/commit/3208df2a100f58569165081cd20c02abed827286)
> 
> Might be good to update the PR description to reflect the changes (and the commit message as well, if you need to repush).
> 
> Suggestion for the PR title: `doc: update IBD requirements in doc/README.md`

@jonatack thanks! Updated the PR title as you suggested. The commit message was created automatically when I clicked the "Commit Suggestion" button on your previous feedback. Fixing it now.

**Comment by Mackain** [comment]
> Please squash your commits according to https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

@maflcko  thanks for pointing that out, fixing now

**Comment by jonatack** [comment]
> The commit message was created automatically when I clicked the "Commit Suggestion" button on your previous feedback. Fixing it now.

@Mackain The commit message wasn't updated yet, could you update it and re-push the same commit (thanks!)

**Comment by Mackain** [comment]
> > The commit message was created automatically when I clicked the "Commit Suggestion" button on your previous feedback. Fixing it now.
> 
> @Mackain The commit message wasn't updated yet, could you update it and re-push the same commit (thanks!)
> 
> Edit: if helpful, reckon you could use the same commit message as the PR title.

Ah, sorry. I now understand what you mean. I originally thought you wanted me to update the commit message of [3208df2](https://github.com/bitcoin/bitcoin/commit/3208df2a100f58569165081cd20c02abed827286), I did that, but then soon after squashed the two commits as per maflcko's comment. I apologize for the confusion and will now update the remaining commit message. Thank you for your patience.

**Review by danielabrozzoni — APPROVED** [review]
ACK 36a6d4b0078ebb39ed082c866bf49214a2a01241

**Review by jonatack — COMMENTED** [review]
ACK 36a6d4b0078ebb39ed082c866bf49214a2a01241

File: doc/README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ Bitcoin Core
 
 Setup
 ---------------------
-Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions, which requires a few hundred gigabytes of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
+Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions, which requires several hundred gigabytes or more of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to several days or more.
```
Comment (ismaelsadeeq) [inline doc/README.md:6]: nit: maybe just say hundreds
```suggestion
Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions, which requires hundreds of gigabytes of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to several days or more.
```

**Review by ismaelsadeeq — COMMENTED** [review]
ACK 36a6d4b0078ebb39ed082c866bf49214a2a01241

File: doc/README.md (hunk 2 above)
Comment (jonatack) [reply doc/README.md:6]: I think I prefer "several hundred or more", as given the current size required, it conveys the floor to expect a bit more, and when the requirement breaks past a terabyte, "or more" allows more time before updating again, whereas "hundreds" might be construed as or implies "not thousands."

Comment (ismaelsadeeq) [reply doc/README.md:6]: > whereas "hundreds" might be construed as or implies "not thousands."

I just wanted to be specific on the what is the requirement currently. 
"several hundred or more" is not specific.

> it conveys the floor to expect a bit more, and when the requirement breaks past a terabyte, "or more" allows more time before updating again.

Fair enough, it is likely to break hundreds, so this comment can be ignored; it's a non-blocking nit.

**Review by tdb3 — APPROVED** [review]
ACK 36a6d4b0078ebb39ed082c866bf49214a2a01241

Thanks. It's good to keep user-facing docs current, to manage user expectations.

**Review by itornaza — APPROVED** [review]
ACK 36a6d4b0078ebb39ed082c866bf49214a2a01241

**Comment by achow101** [comment]
ACK 36a6d4b0078ebb39ed082c866bf49214a2a01241

**Pull request merged**